build/settings.js

```diff
@@ -38,7 +38,8 @@ settings = {
   remoteUrl: 'https://api.resin.io',
   dashboardUrl: 'https://dashboard.resin.io',
   dataDirectory: path.join(userHome, '.resin'),
-  imageCacheTime: 1 * 1000 * 60 * 60 * 24 * 7
+  imageCacheTime: 1 * 1000 * 60 * 60 * 24 * 7,
+  tokenValidityTime: 1 * 1000 * 60 * 60
```

Comment:
The token will still be valid after this time, it's just a good idea to refresh the token regularly so it doesn't expire on an active user